jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -1033,6 +1033,35 @@ void convert_annotations(
   }
 }
 
+/// Convert java annotations, e.g. as retrieved from the symbol table, back
+/// to type annotationt (inverse of convert_annotations())
+/// \param java_annotations: The java_annotationt collection to convert
+/// \param annotations: The annotationt collection to populate
+void convert_java_annotations(
+  const std::vector<java_annotationt> &java_annotations,
+  java_bytecode_parse_treet::annotationst &annotations)
+{
+  for(const auto &java_annotation : java_annotations)
+  {
+    annotations.emplace_back(java_bytecode_parse_treet::annotationt());
+    auto &annotation = annotations.back();
+    annotation.type = java_annotation.get_type();
+
+    std::transform(
+      java_annotation.get_values().begin(),
+      java_annotation.get_values().end(),
+      std::back_inserter(annotation.element_value_pairs),
+      [](const java_annotationt::valuet &value)
+      {
+        auto element_value_pair =
+          java_bytecode_parse_treet::annotationt::element_value_pairt();
+        element_value_pair.element_name = value.get_name();
+        element_value_pair.value = value.get_value();
+        return element_value_pair;
```

Comment:
maybe possible to initialize element_value_pair directly using `element_value_pairt{value.get_name(), value.get_value()}`

Reply:
And then I can just return that directly without creating a variable for it. Thanks, done!